application-templates/starter/package.json

```
@@ -35,18 +35,18 @@
     "@commercetools-frontend/sdk": "21.1.0",
     "@commercetools-test-data/commons": "2.4.1",
     "@commercetools-test-data/core": "2.4.1",
-    "@commercetools-uikit/constraints": "^13.0.2",
-    "@commercetools-uikit/data-table": "^13.0.2",
-    "@commercetools-uikit/flat-button": "^13.0.2",
-    "@commercetools-uikit/grid": "^12.2.9",
-    "@commercetools-uikit/hooks": "^13.0.2",
-    "@commercetools-uikit/icons": "^13.0.2",
-    "@commercetools-uikit/link": "^13.0.2",
-    "@commercetools-uikit/loading-spinner": "^13.0.2",
-    "@commercetools-uikit/notifications": "^13.0.2",
-    "@commercetools-uikit/pagination": "^13.0.3",
-    "@commercetools-uikit/spacings": "^13.0.2",
-    "@commercetools-uikit/text": "^13.0.2",
+    "@commercetools-uikit/constraints": "^14.0.0",
```

```suggestion
    "@commercetools-uikit/constraints": "14.0.0",
```
I am not so sure which we are going with (if there is any issue at all), but I got a [feedback](https://github.com/commercetools/merchant-center-frontend/pull/11941#discussion_r819514596) regarding being specific about the version we are updating. 
What do you think?

---

Good question. This is a different use case though, as it's a starter template.
The point of having version ranges here is to ensure the latest possible uikit versions are installed by default.
People can then decide on their own if they want to pin the versions or not.

Hope it makes sense.

---

Ah, I see. It makes sense to me. 
Thanks for clarifying.